com.unity.shadergraph/Documentation~/Block-Node.md

```diff
@@ -0,0 +1,34 @@
+# Block Node
+
+## Description
+
+A Block is a specific type of Node for the Master Stack. A block represents a single piece of the surface (or vertex) description data used in the final shader output. Some Block nodes are always available, while some may only be available with certain pipelines.  
```

Comment:
I would argue we should either specify "piece" as a shader stage _or_ remove the reference to surface/vertex and just replace it with material/shader ie "a single piece of the material data used in the final shader output"